spring-rabbit/src/main/java/org/springframework/amqp/rabbit/annotation/RabbitListenerAnnotationBeanPostProcessor.java

```diff
@@ -440,14 +432,10 @@ protected Collection<Declarable> processListener(MethodRabbitListenerEndpoint en
 		List<Object> resolvedQueues = resolveQueues(rabbitListener, declarables);
 		if (!resolvedQueues.isEmpty()) {
 			if (resolvedQueues.get(0) instanceof String) {
-				endpoint.setQueueNames(resolvedQueues.stream()
-						.map(o -> (String) o)
-						.collect(Collectors.toList()).toArray(new String[0]));
+				endpoint.setQueueNames(resolvedQueues.stream().map(o -> (String) o).toArray(String[]::new));
```

Comment:
Convention is new line per `.`.

Reply:
I know, but looks a bit awkward as a method argument.
I can bring it back if you wish or introduce a local variable.